src/notifications/selectors.ts

```diff
@@ -0,0 +1,9 @@
+import { State } from './reducer'
+
+const notifications = (state: any): State => state.notifications
+
+export const readNotificationList = state =>
+    notifications(state).readNotificationList
```

Comment:
I really recommend using the `reselect` package to create your selectors more simply, and with auto typings as long as you have the base selector typed (check out selector modules in other features for examples)